models/hourly_spend.sql

```diff
@@ -459,34 +431,6 @@ automatic_clustering_spend_hourly as (
     group by 1, 2, 3, 4
 ),
 
-materialized_view_spend_hourly as (
-    select
-        hours.hour,
-        'Materialized Views' as service,
-        null as storage_type,
-        null as warehouse_name,
-        null as database_name,
-        coalesce(
-            sum(
-                stg_metering_history.credits_used * daily_rates.effective_rate
-            ),
-            0
-        ) as spend,
-        spend as spend_net_cloud_services,
-        any_value(daily_rates.currency) as currency
-    from hours
-    left join {{ ref('stg_metering_history') }} as stg_metering_history on
-        hours.hour = convert_timezone(
-            'UTC', stg_metering_history.start_time
-        )
-        and stg_metering_history.service_type = 'MATERIALIZED_VIEW'
-    left join {{ ref('daily_rates') }} as daily_rates
-        on hours.hour::date = daily_rates.date
-            and daily_rates.service_type = 'MATERIALIZED_VIEW'
-            and daily_rates.usage_type = 'materialized views'
-    group by 1, 2, 3, 4
-),
-
```

Comment:
This change will "rename" this service from `Materialized Views` to `Materialized View`. If that's a problem, we should revert this part.

Reply:
As discussed we're going to keep the 'Materialized Views' name

Reply:
I updated the code to rename it from the new source to `Materialized Views` and tested it with real data